apps/studio/components/layouts/DatabaseLayout/Database.Commands.tsx

```diff
@@ -104,4 +104,166 @@ export function useDatabaseGotoCommands(options?: CommandOptions) {
     ],
     { ...options, deps: [ref] }
   )
+
+  useRegisterCommands(
```

Comment:
since these are all being added to the same section with the same options, you should be able to put them all in the same array in a single hook